bigquery_etl/glam/templates/histogram_bucket_counts_v1.sql

```diff
@@ -46,7 +46,6 @@ RETURNS ARRAY<STRUCT<key STRING, value FLOAT64>> AS (
 CREATE TEMP FUNCTION udf_normalize_histograms(
   arrs ARRAY<
     STRUCT<
-      latest_version INT64,
```

Comment:
This was causing the bucket_counts query to fail, but the failure was not caught because the queries are being run in the background.

Reply:
Issue for followup: https://github.com/mozilla/bigquery-etl/issues/1470